comex-main/src/main/java/br/com/alura/comex/relatorios/RelatorioSintetico.java

```diff
@@ -0,0 +1,64 @@
+package br.com.alura.comex.relatorios;
+
+import java.math.BigDecimal;
+import java.util.*;
+import java.util.function.Function;
+
+import br.com.alura.comex.*;
+
+public class RelatorioSintetico {
+
+	Object totalProdutosVendidos = 0;
+	Object totalDePedidosRealizados = 0;
+	BigDecimal montanteDeVendas = BigDecimal.ZERO;
+	Pedido pedidoMaisBarato = null;
+	Pedido pedidoMaisCaro = null;
+	Object totalDeCategorias = 0;
```

Comment:
Atributos devem ser `private`. Além disso, defina os tipos!

```suggestion
	int totalProdutosVendidos = 0;
	long totalDePedidosRealizados = 0;
	BigDecimal montanteDeVendas = BigDecimal.ZERO;
	Pedido pedidoMaisBarato = null;
	Pedido pedidoMaisCaro = null;
	int totalDeCategorias = 0;
```